frappe/twofactor.py

```diff
@@ -74,6 +74,9 @@ def cache_2fa_data(user, token, otp_secret, tmp_id):
 
 def two_factor_is_enabled_for_(user):
 	'''Check if 2factor is enabled for user.'''
+	if user == "Administrator":
+		return False
```

Comment:
why?

Reply:
on frappe cloud Administrator can not log in, as the admin doesn't have contact so can't send OTP and default email id for admin is admin@example.com